orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/tasks/SadNoOpTask.java

```diff
@@ -0,0 +1,17 @@
+package com.netflix.spinnaker.orca.pipeline.tasks;
+
+import com.netflix.spinnaker.orca.api.pipeline.SkippableTask;
+import com.netflix.spinnaker.orca.api.pipeline.TaskResult;
+import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionStatus;
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SadNoOpTask implements SkippableTask {
+  @NotNull
+  @Override
+  public TaskResult execute(@NotNull StageExecution stage) {
+    return TaskResult.builder(ExecutionStatus.TERMINAL).build();
+  }
+}
```

Comment:
Same goes here, I'd prefer to see this in -nflx rather than polluting OSS.

Reply:
yes, it's for testing purposes. It lives right next to the already existing `NoOpTask`, is that also polluting OSS? I figured it might be useful even when you're testing out a new pipeline, let's say you want to see how a stage failing on a branch impacts your ongoing deploy on a different branch

Reply:
removed in b98bf056e